cli/admin/src/main/java/io/pravega/cli/admin/serializers/controller/ActiveTxnRecordSerializer.java

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
As discussed we should not need the serializers for the READ APIs

Reply:
I have removed these serializers and opted to have the serializer return the record object directly.